package-lock.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should this file be checked in?

Reply:
oh yes.. this would make sure the CI environment installs the right packages via `npm ci`.

Reply:
Ah gotcha. Makes sense!